src/main/scala/tilelink/SRAM.scala

```diff
@@ -24,7 +24,8 @@ class TLRAM(
     beatBytes: Int = 4,
     ecc: ECCParams = ECCParams(),
     val devName: Option[String] = None,
-  )(implicit p: Parameters) extends DiplomaticSRAM(address, beatBytes, devName)
+    val compatName: String = "sifive,sram0"
```

Comment:
Thinking about this more, I'm wary of having the same default value set at multiple different levels, since it makes it harder to tell which level of abstraction the actual value came from. Instead, what if you made this an `Option[Seq[String]]` which defaults to `None` both here and in `DiplomaticSRAM`, and within the body of `DiplomaticSRAM`, you do a `new SimpleDevice(_, compatible.getOrElse(Seq("sfive,sram0")))`?